pkg/strutil/convert.go

```diff
@@ -24,3 +25,19 @@ func NoCopyBytesToString(b []byte) string {
 func NoCopyStringToBytes(s string) []byte {
 	return *(*[]byte)(unsafe.Pointer(&s))
 }
+
+func ToStrSlice[T any](input []T, withQuote ...bool) []string {
```

Comment:
add ut here

Reply:
test: int, string with single quote, byte, and others

Reply:
ok